src/styles/index.scss

```diff
@@ -2,2 +1,86 @@
+@import './variables';
+@import './card';
+@import './stars';
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --active-color: #00acdc;
+}
+
+.body {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav {
+  margin: 0 50px 0 0;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+
+  color: #060b35;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+li {
+  margin-right: 20px;
+}
```

Comment:
Direct styling of `li` elements can be risky if `li` elements are used elsewhere outside `.nav__list`. It's better to use a class selector to avoid styling conflicts.